lib/polling.ts

```diff
@@ -1,42 +1,54 @@
 import { HubSpotPromise } from '@hubspot/local-dev-lib/types/Http';
-import { ValueOf } from '@hubspot/local-dev-lib/types/Utils';
-import { POLLING_DELAY, POLLING_STATUS } from './constants';
+import { DEFAULT_POLLING_DELAY } from './constants';
+
+export const DEFAULT_POLLING_STATES = {
+  STARTED: 'STARTED',
+  SUCCESS: 'SUCCESS',
+  ERROR: 'ERROR',
+  REVERTED: 'REVERTED',
+  FAILURE: 'FAILURE',
+} as const;
+
+const DEFAULT_POLLING_STATUS_LOOKUP = {
+  successStates: [DEFAULT_POLLING_STATES.SUCCESS],
+  errorStates: [
+    DEFAULT_POLLING_STATES.ERROR,
+    DEFAULT_POLLING_STATES.REVERTED,
+    DEFAULT_POLLING_STATES.FAILURE,
+  ],
+};
 
 type GenericPollingResponse = {
-  status: ValueOf<typeof POLLING_STATUS>;
+  status: string;
 };
 
-type PollingCallback<T extends GenericPollingResponse> = (
-  accountId: number,
-  taskId: number | string
-) => HubSpotPromise<T>;
+type PollingCallback<T extends GenericPollingResponse> =
```

Comment:
I changed the signature of the callback so we could simplify the arguments in `poll`. This makes the polling function more generic so it can be more easily leveraged in other places.